src/os_posix/os_fs.c

```diff
@@ -422,10 +422,25 @@ __posix_file_read(
     /* Break reads larger than 1GB into 1GB chunks. */
     for (addr = buf; len > 0; addr += nr, len -= (size_t)nr, offset += nr) {
         chunk = WT_MIN(len, WT_GIGABYTE);
+#ifdef RHEL_PPC
+        if ((nr = pread(pfh->fd, addr, chunk, offset)) <= 0) {
+            /* Retry as the RHEL PPC variants we use have false failures that occur on pread. */
+            __wt_verbose(session, WT_VERB_READ,
+              "%s: handle-read: pread: failed to read %" WT_SIZET_FMT " bytes at offset %" PRIuMAX
+              ", will retry",
+              file_handle->name, chunk, (uintmax_t)offset);
+            if ((nr = pread(pfh->fd, addr, chunk, offset)) <= 0)
```

Comment:
This feels somewhat similar to the WT_SYSCALL_RETRY macro - could we reuse that instead?